README.md

```diff
@@ -1,7 +1,70 @@
+[![License][badge-apache-license]](https://opensource.org/licenses/Apache-2.0)
 [![Coverage][badge-coverage]][badge-url-coverage]
+[![Python 3.6][badge-python]](https://www.python.org)
 
-# tes-compliance-suite
-An application allowing the TES servers to test their conformance to TES Specifications
+# TES Compliance Suite
 
+The TES Compliance Suite determines a server's compliance with the [TES API specification][res-tes-spec]. The specification has been developed by the [Global Alliance for Genomics and Health][res-ga4gh], an international coalition, formed to enable the sharing of genomic and clinical data. It serves to provide a standardized API framework and data structure to allow for interoperability of datasets hosted at different institutions.
+
+## Description
+
+The compliance suite is designed as an abstract and API specification independent runner. It allows reusability of specs without having to make significant code changes. The suite is run via multiple YAML-based test files defining the test flow of an API endpoint.
+
+## Documentation
+
+Please refer [Installation and Usage][res-doc-installation]  to get started with the installation and usage of suite.
+
+You can also find more details about suite architecture and methodology.
+
+- [Preamble](https://github.com/elixir-cloud-aai/tes-compliance-suite/blob/dev/docs/preamble.md)
```

Comment:
Yes, I definitely agree. The reason I switched to URL instead of a relative path is, that when we publish a package to PyPI, it doesn't understand the relative paths. For images, we need to put raw URL links (even permalinks don't work). And for the Markdown files which cant be put as raw, we need to put the URL pointing to a commit/branch.

Please suggest what we should use. Using the relative path will cause the PyPI page to break and look ugly.

Reply:
Permalinks have the problem that they will never update if you don't specifically remember to fix them. E.g., we update the logos - it won't show on the `README.md` unless you update the link.

Links to specific branches have the disadvantage that they may break, and that, if you change them, you won't see what they behave like until you merge them into the targeted branch.

I'd say, screw PyPI and use relative links :) Image links also work when they are relative. Check here for example: https://raw.githubusercontent.com/elixir-cloud-aai/foca/dev/README.md

Reply:
I was looking further into PyPI. It seems the file retrieval process for PyPI uses `pkg_resources`. This will be a significant change codewise. I think we can push to PyPI later.

Reply:
Yes, I think pushing to PyPI is something that makes most sense to do _after_ the runner and the tests have been separated.

Reply:
Right, that will be better.